src/main/kotlin/org/rust/lang/core/completion/CompletionEngine.kt

```diff
@@ -35,6 +40,7 @@ fun RsCompositeElement.createLookupElement(scopeName: String): LookupElement {
         is RsFunction -> base
             .withTypeText(retType?.typeReference?.text ?: "()")
             .withTailText(valueParameterList?.text?.replace("\\s+".toRegex(), " ") ?: "()")
+            .appendTailText(generateAppendTailText(), false)
```

Comment:
Don't you think it'll look better with `grayed = true`? Won't distract from the function signature so much.